validphys2/src/validphys/process_options.py

```diff
@@ -279,16 +291,18 @@ def _dyncpt_xq2map(kin_info):
     xq2map_function=_displusjet_xq2map,
 )
 
-DY_W_ETA = _Process(
-    "DY_W_ETA",
-    "DY W -> l nu pseudo rapidity",
+
+DY_NC = _Process(
```

Comment:
I would call this `DY_2_LL` or `DY_2L` or something like that since the same kinematics can be used for both Z and W.

If you want to separate between `DY_NC` and `DY_CC` I would use those names directly. As this stands this is "wrong" since it says `DY_NC` but in the description it says that it can be either W or Z.

So I suggest you either use a generic name and then the process type is the same for both datasets, or if you want to differentiate, you can create `DY_NC` and `DY_CC` (both using the same function `_dyboson_xq2map`)

Reply:
I'm slightly puzzled because in this branch I got rid of the process `DY_W_ETA` some commits ago. Now I only use `DY_NC`.

Reply:
Ok I understand.